bigquery/deploy/bq-indexer.yaml.templ

```diff
@@ -7,9 +7,16 @@ spec:
     metadata:
       name: bq-indexer
     spec:
+      nodeSelector:
+        type: indexer
       containers:
       - name: bq-indexer
         image: gcr.io/PROJECT_ID/bq-indexer
+        resources:
```

Comment:
This resources section can be removed, right -- the indexer doesn't need that much cpu, right?

Reply:
Sorry I mean just set req